src/login/styles/_Notifications.scss

```diff
@@ -70,4 +152,43 @@
       width: calc(100% - 1.25rem);
     }
   }
-}
\ No newline at end of file
+}
+
+@media (max-width: 432px) {
+  .speech-bubbletip.settings,
+  .speech-bubbletip.identity {
+    width: 270px;
+    white-space: normal;
+    top: -65px;
+  }
+  .speech-bubbletip.identity.short {
+    top: -52px;
+  }
+  .speech-bubbletip.settings.en {
+    left: -80px;
+  }
+  .speech-bubbletip.settings.sv {
+    left: 5px;
+  }
+  .speech-bubbletip.identity {
+    left: -56px;
+  }
+  .speech-bubbletip.identity::before {
+    left: 70%;
+  }
+  .speech-bubbletip.settings.en::before {
+    left: 80%;
+  }
+  .speech-bubbletip.settings.sv::before {
+    left: 48%;
+  }
+}
+
+@media (max-width: 384px) {
+  .speech-bubbletip.settings.en{
+    left: 55px;
+  }
+  .speech-bubbletip.settings.en::before {
+    left: 30%;
+  }
+}
```

Comment:
A question I would ask myself building something with a lot of custom code would be what the impact of changing the text in the future would be and if there was anything more I could do to prepare for the next person.

Reply:
This was the solution I could find without getting the text to go outside the browser sizes in responsive.
I would appreciate any different solutions for this issue, if you have any good ideas.